### PR TITLE
[FEATURE] Implements "Hedy's choice" to set favorites on /explore page

### LINKED ISSUE #2521: [UI idea] Implement 'Hedy choice' program and highlight on /explore page
**Idea incl level**
It would be nice if we enable the admin to select a program and highlight this one as the `Hedy choice` program. Which is shown as the first one on the explore page and has a differentiating color. The admin is free to switch this any time by simply selecting one on the admin page.

### PR DESCRIPTION
**Pull request by TiBiBa**
**Description**
In this PR we implement a new and exciting feature: Hedy's choice! Enabling the admin to set programs as favorite and show these at the top of the /explore page. When filtering the favorite programs are hidden to prevent overly complex design. The programs are marked with a `hedy_choice` boolean on the Programs table, filtered with an index key. The view is limited to the 4 most recent programs. The functionality is written to retrieve all, which might be useful later on to allow the admin to change the favorite programs on the admin page as well.

**Fixes**
This PR fixes #2521

**How to test**
Make sure you are an admin and navigate to the `/explore` page. Verify that the UI has changed and now each program container has a small "star button" enabling you to select/remove a program as favorite. The color should change dynamically when selecting accompanied by a modal.alert. For the actual "favorite box", reload the explore page.

**Screenshot**
Schermafbeelding 2022-05-10 om 13 33 06

### CODE REVIEW

File: static/js/app.ts [hunk 1]
```diff
@@ -711,6 +711,36 @@ export function submit_program (id: string, index: number) {
   });
 }
 
+export function set_explore_favourite(id: string, favourite: number) {
+  let prompt = "Are you sure you want to remove this program as a \"Hedy choice\" program?";
```
Comment (Felienne) [inline static/js/app.ts:715]:
```suggestion
  let prompt = "Are you sure you want to remove this program as a \"Hedy\'s choice\" program?";
```

File: static/js/app.ts [hunk 2]
```diff
@@ -711,6 +711,36 @@ export function submit_program (id: string, index: number) {
...
+  if (favourite) {
+    prompt = "Are you sure you want to set this program as a \"Hedy choice\" program?";
```
Comment (Felienne) [inline static/js/app.ts:717]:
```suggestion
    prompt = "Are you sure you want to set this program as a \"Hedy\'s choice\" program?";
```

**Review by Felienne — CHANGES_REQUESTED** [review]
Works!

Can we just make Hedy Choice in the popup into Hedy's Choice? (I am not 100% sure my suggested commits are correct)

**Comment by TiBiBa** [comment]
> Works!
> 
> Can we just make Hedy Choice in the popup into Hedy's Choice? (I am not 100% sure my suggested commits are correct)

Did you already commit these changes? Will fix the merge conflicts so we are ready to merge!

**Comment by Felienne** [comment]
> > Works!
> > Can we just make Hedy Choice in the popup into Hedy's Choice? (I am not 100% sure my suggested commits are correct)
> 
> Did you already commit these changes? Will fix the merge conflicts so we are ready to merge!

Yes! It seems to all be fine but I am a bit afraid changing ts form the browser :)

**Review by Felienne — APPROVED** [review]

**Pull request merged**